pkg/services/ldap/settings.go

```diff
@@ -135,9 +138,28 @@ func readConfig(configFile string) (*Config, error) {
 		}
 	}
 
+	for _, server := range result.Servers {
+		envValue := evalEnvVar(server.BindPassword)
+		server.BindPassword = envValue
+	}
+
 	return result, nil
 }
 
+func evalEnvVar(value string) string {
```

Comment:
Heh, I would use a dependency, but for that small thing, I think we should be good for now. But if there is more complicated things intended I would recommend to use some sort of template engine :)

Reply:
I think regexp is just about right for something like this.